src/assets/js/archive.js

```diff
@@ -146,16 +146,51 @@ function updateDownloadLink(releases, os, arch) {
       $('.apple-silicon').each(function(){
         this.remove();
       })
-      
+
       return;
     }
-    
+
     updateReleaseDownloadButton(releasesForArm64, releases.base_url, os, 'arm64');
   } else {
     updateReleaseDownloadButton(releasesForChannel, releases.base_url, os);
   }
 }
 
+/**
```

Comment:
@parlough : Could we make this its own JS file so we could use it elsewhere, like getting started?